trollflow2/__init__.py

```diff
@@ -122,10 +123,10 @@ def save_datasets(job):
         filename = compose(os.path.join(outdir, fname_pattern), fmat)
         fmat.pop('format', None)
         try:
-            objs.append(scns[fmat['areaname']].save_dataset(fmat['productname'], filename=filename, compute=False, **fmat))
+            objs.append(scns[fmat['area']].save_dataset(fmat['product'], filename=filename, compute=False, **fmat))
         except KeyError as err:
             LOG.info('Skipping %s: %s', fmat['productname'], str(err))
-        fmat_config['filename'] = filename
```

Comment:
Is this added intendation intentional? It removes the filenames from the product list `formats` list.